fuse-overlayfs.1

```diff
@@ -99,6 +104,18 @@ $ stat \-c %u:%g merged/a merged/b
 .PP
 Those are the same IDs visible from outside the user namespace.
 
+.PP
+\fB\-o squash\_to\_root\fP
```

Comment:
Why have two different options for this?  Why not squash_to_uid=0?

Reply:
we already have it in a released version, so it would be a breaking change.

I can make `squash_to_root` obsolete though and we can remove it in the future

Reply:
SGTM